src/main/java/com/amihaiemil/versioneye/RtOrganization.java

```diff
@@ -35,6 +72,37 @@ public Projects projects() {
 
     @Override
     public String name() {
-        return null;
+        return this.organization.getString("name");
+    }
+
+    @Override
+    public String company() {
+        String company;
+        if(this.organization.isNull("company")) {
+            company = null;
+        } else {
+            company = this.organization.getString("company");
+        }
+        return company;   
```

Comment:
@SherifWaly Here you can simply say ``return this.organization.getString("company", "")`` -- it will return an empty string if the company attribute is null. We want to avoid returning nulls as much as possible :D